### ISSUE TITLE
Edit relationship field modal closes immediately in PHP 5.6 if using Gutenberg editor

### ISSUE THREAD

**Issue by andrewmondy** [pods-framework/pods#5921]
## Issue Overview
Edit relationship field modal closes immediately after opening after upgrading to PHP 5.6 and using Gutenberg (REST API) editor

## Expected Behavior
Modal should remain open and allow edits.

## Current Behavior
Modal opens for a brief moment and closes before fully loading. Note: Add New modal works, only Edit fails.

## Steps to Reproduce (for bugs)
1. Upgrade to PHP 5.6
2. Enable REST API to use Gutenberg on Pod
3. Click Edit icon in relationship custom field (List View)

## WordPress Environment

<details>
```
WordPress Version: 5.6

PHP Version: 7.4.13

MySQL Version: 5.7.31

Server Software: Apache

Your User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 11_1_0) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/87.0.4280.141 Safari/537.36

Session Save Path: /var/lib/php/sessions

Session Save Path Exists: Yes

Session Save Path Writeable: Yes

Session Max Lifetime: 1440

Opcode Cache:

Apc: No
Memcached: No
OPcache: Yes
Redis: No
Object Cache:

APC: No
APCu: No
Memcache: No
Memcached: Yes
Redis: Yes
WPDB Prefix: wp_

WP Multisite Mode: No

WP Memory Limit: 40M

Current Memory Usage: 4.873M

Current Memory Usage (real): 6.000M

Pods Network-Wide Activated: No

Pods Install Location: /nas/content/live/smfdpidev/wp-content/plugins/pods/

Pods Tableless Mode Activated: No

Pods Light Mode Activated: No

Currently Active Theme: Twenty Twenty

Currently Active Plugins:

Pods - Custom Content Types and Fields: 2.7.26

```
</details>

## Pods Package Export (helpful!)

```
Copy and Paste the JSON Export from **Pods Admin, Migrate: Packages, Export** in WordPress admin here
```

**Comment by JoryHogeveen** [comment]
Hi @andrewmondy 

I cannot reproduce this behavior locally, maybe a plugin conflict?
https://docs.pods.io/faqs/plugin-theme-conflicts/

**Comment by andrewmondy** [comment]
Hi @JoryHogeveen,

No other plugins are installed. This is a fresh WP instance running on WPEngine. The only changes are:

Upgrading to WP 5.6

Installing Pods plugin

Create a parent Pod, and a child Pod, with one single-select relationship field on each, and make it bi-directional. 

Enable REST API on the child to allow Gutenberg editor.

If I add a new Child record from the Parent record, it edits properly in Gutenberg. If I edit the new Child record, the modal immediately closes, unless I disable REST API.

Screen Recording Video:
https://youtu.be/iEYVqKiNi5I

**Comment by JoryHogeveen** [comment]
Hi @andrewmondy 

Ah right, sorry, I missed the Gutenberg part within the modal.
I can reproduce this behavior and will mark it as a bug!

**Comment by JoryHogeveen** [comment]
@sc0ttkclark 
Let me know what you think about this issue. I know 2.8 is very far and fully removed the current DFV implementation.. a bugfix for this could be a waste if 2.8 is almost ready for release.

**Comment by JoryHogeveen** [comment]
@sc0ttkclark Close/punt in favor of 2.8?

**Comment by sc0ttkclark** [comment]
I think this would be difficult to fix in 2.7.x as this area of the code has been refactored for React in 2.8 already. I’m going to see if we can get this fixed in 2.8 if it’s still happening there.

**Comment by zrothauser** [comment]
@sc0ttkclark Looks Relationship fields are working fine in 2.8.
